NEWS.md

```diff
@@ -1,4 +1,18 @@
-## 2.2.0 - Unreleased
+## 2.3.0 - Unreleased
+
+## 2.2.0 - Released (Quesnelia R1 2024)
+The primary focus of this release was error handling improvements and spring version upgrade
+
+[Full Changelog](https://github.com/folio-org/mod-ebsconet/compare/v2.1.0...v2.2.0)
+
+### Stories
+* [MODEBSNET-70] (https://folio-org.atlassian.net/browse/MODEBSNET-70) - Add error code if default note type "General note" is not exists
+
+### Bug Fixes
+* [MODEBSNET-66] (https://folio-org.atlassian.net/browse/MODEBSNET-66) - Add possibility to work with empty vendorDetails object
```

Comment:
this already was mentioned in previous release https://github.com/folio-org/mod-ebsconet/releases/tag/v2.1.1
so we can delete from here

Reply:
Is `Full Changelog` section correct then? Should we compare with 2.1.0 or 2.0.0?

Reply:
full changelog we should not change, it's differences between releases.

Reply:
here is fix version 2.1.1 https://folio-org.atlassian.net/browse/MODEBSNET-66

Reply:
fixed